firehose-source/README.md

```diff
@@ -0,0 +1,47 @@
+Spring Cloud Stream Firehose Sample
+=============================
```

Comment:
Could you rephrase this README to have it conform to _e.g._ this template: https://github.com/spring-cloud/spring-cloud-stream-modules/blob/c3d654c2a635474a3f160f62bca0460f91864c0c/counter-sink/README.adoc

Avoid the use of "sample", and simplify the instructions to the maximum.
Use of asciidoc vs markdown would be prefered too.